test/fuzzing/chrono-timepoint.cc

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2021, Paul Dreik
+// For license information refer to format.h.
+#include <fmt/chrono.h>
+
+#include "fuzzer-common.h"
+
+/*
+ * a fuzzer for the chrono timepoints formatters
+ * C is a clock (std::chrono::system_clock etc)
+ */
+template <typename C> void doit(const uint8_t* data, size_t size) {
+  using D = typename C::duration;
+  using TP = typename C::time_point;
+  using Rep = typename TP::rep;
```

Comment:
I suggest inlining these aliases to improve readability since they are only used once or, in the case of Rep, twice. Or just keep `Rep` and inline the other two.